lib/Jit/LLILCJit.cpp

```diff
@@ -394,3 +395,107 @@ void __cdecl LLILCJit::fatal(int Errnum, ...) {
 void LLILCJit::signalHandler(void *Cookie) {
   // do nothing
 }
+
+template <typename ObjSetT, typename LoadResult>
```

Comment:
As a rule, template function definitions should be in headers rather than cpp, no?